main.nf

```diff
@@ -67,49 +68,57 @@ if (params.help) {
     exit 0
 }
 
-
 //Generate empty channels for peptides, proteins and variants
 ch_peptides = Channel.empty()
+ch_check_peptides = Channel.empty()
 ch_proteins = Channel.empty()
 ch_split_variants = Channel.empty()
 
+if ( params.show_supported_models && (params.peptides || params.proteins || params.input || params.proteome)){
```

Comment:
as suggested below we could just run it and exit?

Reply:
see below